gpii/node_modules/flowManager/src/UserLogonStateChange.js

```diff
@@ -30,33 +30,53 @@ gpii.flowManager.userLogonStateChange.handleResetToken = function (activeToken,
     }
 };
 
-gpii.flowManager.userLogonStateChange.errorResponse = function (request, message, statusCode) {
+gpii.flowManager.userLogonHandling.errorResponse = function (request, message, statusCode) {
     request.events.onError.fire({
         statusCode: statusCode || 500,
         message: message || "Unknown error occured on logon change action"
     });
 };
 
-gpii.flowManager.userLogonStateChange.startLifecycle = function (lifecycleManager, lifecyclePayload, event) {
-    gpii.logFully("userLogonStateChange.startLifecycle got final payload ", gpii.renderMegapayload(lifecyclePayload));
+gpii.flowManager.userLogonHandling.startLifecycle = function (that, lifecycleManager, lifecyclePayload, event) {
+    gpii.logFully("userLogonHandling.startLifecycle got final payload ", gpii.renderMegapayload(lifecyclePayload));
     lifecycleManager.start(lifecyclePayload,
         gpii.flowManager.logAndNotify("Lifecycle manager startLifecycle returned: ", event, function () {
+            gpii.flowManager.userLogonHandling.updateLogonChangeModel(lifecycleManager, lifecyclePayload.userToken, "login", false);
             return "User with token " + lifecyclePayload.userToken + " was successfully logged in.";
-        }
-    ));
+        })
+    );
 };
 
-gpii.flowManager.userLogonStateChange.logoutUser = function (request, lifecycleManager, userToken) {
+/**
+ * Updates the `logonChange` portion of the lifecycle manager model.
+ *
+ * @param lifecycleManager {Object}: The lifecycleManager component
```

Comment:
Just for consistency, in case we ever develop a tool that parses these - elsewhere we don't use the ":" after the type annotation, should remove these globally